x-pack/test/plugin_functional/test_suites/global_search/global_search_providers.ts

```diff
@@ -87,7 +87,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     describe('Applications provider', function () {
       it('can search for root-level applications', async () => {
         const results = await findResultsWithApi('discover');
-        expect(results.length).to.be(1);
+        expect(results.length).to.be(2);
```

Comment:
This test appears to be matching the new _Attack discovery_ feature added in this PR